Makefile

```diff
@@ -640,9 +641,13 @@ generate-sqlx-data:  # Regenerate sqlx-data.json based on queries made in Rust c
 dist/firecracker_kernel.tar.gz: firecracker/generate_firecracker_kernel.sh | dist
 	./firecracker/generate_firecracker_kernel.sh
 
+.PHONY: dist/plugin-bootstrap-init
+dist/plugin-bootstrap-init:  ## Build the Plugin Bootstrap Init (+ associated files) and copy it to dist/
+	$(DOCKER_BUILDX_BAKE_HCL) plugin-bootstrap-init
+
 # Changes to any of these files will trigger a rebuild of the rootfs
 FIRECRACKER_ROOTFS_FILES := $(shell find firecracker/packer -type f)
-dist/firecracker_rootfs.tar.gz: $(FIRECRACKER_ROOTFS_FILES) | dist
+dist/firecracker_rootfs.tar.gz: $(FIRECRACKER_ROOTFS_FILES) dist/plugin-bootstrap-init | dist
```

Comment:
fwiw the main reason I don't have true dependencies for plugin-bootstrap-init is because it's difficult to express the full Rust build reqs in Make-ish (toplevel cargo.toml, Dockerfile, src/plugin-bootstrap, etc...).

I don't love it but, is what it is I guess

Reply:
yup... this is what we have to deal with, unfortunately 😢